Jenkinsfile

```
@@ -1,105 +1,5 @@
-        docker.image("${dockerImage}-alpine").inside('-v /var/run/docker.sock:/var/run/docker.sock') {
-          stage('sonarqube') {
-            // push analysis results to sonar
-            // do not use --parallel, as sonarqube can not work in this mode
-            sh(script: "#!/bin/sh\n./gradlew sonarqube -x test --configure-on-demand \
-              -Dsonar.host.url=https://sonar.soramitsu.co.jp \
-              -Dsonar.login=${SONAR_TOKEN}")
-          } // end stage
-          stage('build and push docker image') {
-            sh """#!/bin/sh
-              apk update && apk add docker
-              docker login --username ${DH_USER} --password '${DH_PASS}' https://${DH_REPO_URL}
-              ./gradlew dockerPush -x test -PrepoUrl=${DH_REPO_URL}
-            """
-            // push git_tag
-            print "Info: docker_push_tags=${docker_push_tags}"
-            for ( git_tag in docker_push_tags ){
-              sh """#!/bin/sh
-                apk update && apk add docker
-                docker login --username ${DH_USER} --password '${DH_PASS}' https://${DH_REPO_URL}
-                ./gradlew dockerPush -x test -PrepoUrl=${DH_REPO_URL} -Pversion=${git_tag}
-              """
-            } //end for
-          } // end stage
-        } // end docker
-      } // end if
-    } // end withCredentials
-  } // end try
-  catch(Exception e) {
-    currentBuild.result = 'FAILURE'
-    print e
-  } // end catch
-  finally {
-    stage('upload test artifacts') {
-      junit allowEmptyResults: true, keepLongStdio: true, testResults: '**/build/test-results/**/*.xml'
-      jacoco execPattern: '**/build/jacoco/test.exec', sourcePattern: '**/src/main/java'
-    } //end stage
-    stage('send email notification') {
-      emailSubject = '${PROJECT_DISPLAY_NAME} :: ${BUILD_STATUS} :: #${BUILD_NUMBER}'
-      emailBody =
-        '''
-          Build status: ${BUILD_STATUS}<br/>
-          Job: ${PROJECT_NAME}<br/>
-          Build: #${BUILD_NUMBER}<br/>
-          ${FAILED_TESTS}<br/>
-          More info at: ${BUILD_URL}<br/><br/>
-          --------LOG-BEGIN--------<br/>
-          <pre style='line-height: 22px; display: block; color: #333; font-family: Monaco,Menlo,Consolas,"Courier New",monospace; padding: 10.5px; margin: 0 0 11px; font-size: 13px; word-break: break-all; word-wrap: break-word; white-space: pre-wrap; background-color: #f5f5f5; border: 1px solid #ccc; border: 1px solid rgba(0,0,0,.15); -webkit-border-radius: 4px; -moz-border-radius: 4px; border-radius: 4px;'>
-            ${BUILD_LOG, maxLines=50, escapeHtml=true}
-          </pre>
-          --------LOG-END--------
-      '''
-      emailext body: "${emailBody}", replyTo: "${replyTo}", subject: "${emailSubject}", to: "${sendTo}"
-    } //end stage
-    cleanWs()
-  } // end finally
-} //end node
+new org.soramitsu.mainLibrary().call(
+  registry: 'docker.soramitsu.co.jp'
```

```suggestion
  registry: 'docker.soramitsu.co.jp',
  nexusUserId: "bot-sora-rw"
```

add `bot-sora-rw` to jenkins: https://jenkins.soramitsu.co.jp/job/sora/credentials/

---

Done